### ISSUE TITLE
Python step 14 Sudoku Solver extremely unclear directions 

### ISSUE THREAD

**Issue by a2937** [freeCodeCamp/freeCodeCamp#53047]
### Describe the Issue

The wording is too hard to follow as of now.  It's extremely difficult to understand what to actually do. My first instinct is to build a nested for loop and to try joining the part array.  Only to get confused by what it wants us to do with `str(item)`. Pictured in the code section is what my instinct said to do.  

### Affected Page

https://www.freecodecamp.org/learn/scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/step-14

### Your code

```
      for square_no, part in enumerate([line[:3], line[3:6], line[6:]], start=1):
                for item in part:
                    "|".join(part)
                    str(item)
```

### Expected behavior

Something about putting the for loop inside the join statement would be extremely helpful. Additionally placing `str(item)` in front of the loop where it is declared. 

```PY
 for square_no, part in enumerate([line[:3], line[3:6], line[6:]], start=1):
               "|".join(str(item) for item in part)
```

Nothing about this shortcut makes much sense without more context than the directions provide. 

### System

- Device: Desktop
- OS: Windows 10
- Browser: Firefox
- Version: 121.0

### Additional context

Numerous campers have had issues with it and even I wouldn't know how to solve it if I wasn't aware of the Github source.

https://forum.freecodecamp.org/t/learn-classes-and-objects-by-building-a-sudoku-solver-step-14/662473

https://forum.freecodecamp.org/t/learn-classes-and-objects-by-building-a-sudoku-solver-step-14/662463

https://forum.freecodecamp.org/t/learn-classes-and-objects-by-building-a-sudoku-solver-step-14/660420

**Comment by naomi-lgbt** [comment]
@Dario-DC are we tracking this in that issue thread? Should this be closed, or do we want to determine a resolution and open this up for contribution?

**Comment by Dario-DC** [comment]
@naomi-lgbt Currently we have an umbrella issue for each Python project. Zaira is working on this specific project and she already drafted a PR.

**Comment by ilenia-magoni** [comment]
closing now that the Sudoku Solver has been changed

**Comment by a2937** [comment]
I apologize but I don't remember which PR resolves step 14. May I look at it?

**Comment by Dario-DC** [comment]
> I apologize but I don't remember which PR resolves step 14. May I look at it?

The project has been completely changed.  This part is not there anymore.

**Comment by kimegarar** [comment]
I'm doing it and still is extremely unclear the directions;I'm stuck at step 41 for two days now ·_·'
